docs/implementation-guidelines.md

```diff
@@ -14,37 +14,37 @@ Module repositories for IBM Cloud must use the three-part name format `terraform
 
 Module repository can have nested modules:
 - When naming the nested modules, do not repeat the name of the top level module.
-- For example, for a top level module named `terraform-ibm-cos`, do not name the nested module `cos-instance` (since, it will translate to "cos_cos-instance" in the module registry);  instead, use `instance` (to register the child-module as "cos_instance").
+- For example, for a top level module named `terraform-ibm-cos`, do not name the nested module `cos-instance` (because it will translate to "cos_cos-instance" in the module registry); instead, use `instance` (to register the child-module as "cos_instance").
 
-## Module documentation
+## Module documentation :id=imp-gls-docs
 
 Module documentation should be located in the `README.md` file at the same level as the `main.tf` file for each of the modules in a repository, including the example modules. Additional documentation may be located in a sub directory named `docs`.
```

Comment:
"Additional documentation may be located in a sub directory named `docs`." + and linked from the top level README.md file

Reply:
Each of the examples are actually standalone terraform modules - this is what I am meaning by "example module".